main.go

```diff
@@ -109,3 +110,11 @@ func authType(repo string, username string, token string) transport.AuthMethod {
 
 	return auth
 }
+
+func checkRefs(upstreamRef string, downstreamRef string) (string, string) {
```

Comment:
This whole function shouldn't be needed as the default value for both flags, if not used, is already master.
As I mentioned before the third input is the default value. This is already master.